src/aui-layout/tests/unit/js/aui-layout-builder-resize-col-tests.js

```diff
@@ -723,6 +723,16 @@ YUI.add('aui-layout-builder-resize-col-tests', function(Y) {
             Assert.areEqual(false, dragHandle.item(0).hasClass('layout-builder-resize-col-draggable-visible'));
             Assert.areEqual(true, dragHandle.item(1).hasClass('layout-builder-resize-col-draggable-visible'));
             Assert.areEqual(true, dragHandle.item(2).hasClass('layout-builder-resize-col-draggable-visible'));
+        },
+
+        'should low light resize column handle on mouseout': function() {
```

Comment:
This test name isn't making much sense. After **should** there should always be a verb, otherwise it's wrong English. The idea of test cases using **should** is that the subject is always the class you're testing, so you should make sure that the test name is a phrase that explains something that this class should do.

In this case something like **should remove dragging css class after mouseup event** would work well for example.